helix-core/src/test/java/org/apache/helix/integration/rebalancer/WagedRebalancer/TestWagedRebalance.java

```diff
@@ -164,7 +214,7 @@ public void testWithInstanceTag() throws Exception {
     validate(_replica);
   }
 
-  @Test(dependsOnMethods = "test")
+  @Test(dependsOnMethods = "testRebalanceTool")
```

Comment:
nit, but why the other tests need to depend on testRebalanceTool?

Reply:
Doesn't have to depend on testRebalanceTool, but I like to enforce strict ordering for the tests I write..

Reply:
I would prefer to configure the tests in a way of real logic dependencies. Easier for us to maintain.
Moreover, I think we won't remember why it was initially done like this. Let do it in a meaningful way the first time.

Reply:
Okay. I reverted this change.